example/alertmanager-service.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: v1
 kind: Service
 metadata:
   name: alertmanager-main
+  labels:
+    app: alertmanager
+    instance: main
```

Comment:
The same for the Promtetheus? Like this `app: prometheus` and `prometheus: main`?